metric_metadata/cassandra/cassandra.go

```diff
@@ -35,6 +35,7 @@ type CassandraMetricMetadataConfig struct {
 // NewCassandraMetricMetadataAPI creates a new instance of API from the given configuration.
 func NewCassandraMetricMetadataAPI(config CassandraMetricMetadataConfig) (api.MetricMetadataAPI, error) {
 	clusterConfig := gocql.NewCluster()
+	clusterConfig.Consistency = gocql.One
```

Comment:
i think we should use quorum, with replication_factor set to 3 (or even 5!) we should be safe.